Makefile

```diff
@@ -207,9 +207,6 @@ buildah-dev-push: build
 buildah-push: buildah-build
 	$(SUDO_CMD) buildah push ${IMG}
 
-$(GOPATH)/bin/mockgen:
-	go get -u github.com/golang/mock/mockgen/...
```

Comment:
Could you add this to the developing docs, we don't mention mockgen there at all today.

Reply:
Will do.